pkg/util/mount/nsenter_mount.go

```diff
@@ -87,9 +87,10 @@ func NewNsenterMounter() *NsenterMounter {
 var _ = Interface(&NsenterMounter{})
 
 const (
-	hostRootFsPath     = "/rootfs"
-	hostProcMountsPath = "/rootfs/proc/1/mounts"
-	nsenterPath        = "nsenter"
+	hostRootFsPath        = "/rootfs"
```

Comment:
Thinking about this, is it possible to hide this prefix behind an helper method which does a `stat()` first and returns `/` as a fallback if `/rootfs` doesn't exist (possibly cached)? As `nsenter` does a setns+fchdir+fchroot internally, this will cover containerized, chrooted and unconstrained kubelet the same way.